### PR TITLE
Deploy scripts for XVSVault and GovernorBravo

### PR DESCRIPTION
**Pull request by defistar**
## Description

Deployment scripts for 

XVSVault
XVSVaultProxy
GovernorBravoDelegator
GovernorBravoDelegate

Contract Updates:

- add `proposalCount` to constructor of `GovernorBravoDelegator`
- add `proposalCount` to initialise function of `GovernorBravoDelegate`
- 

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Thenewguy911** [comment]
Why is this not finished after 6 months?  Why is there only one lead Dev and two part time devs working on this?  There is 4 billion in tvl but three developers.

File: script/deploy/xvs-vault/05-set-new-owner-to-xvs-store.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const [network] = args;
+const contractConfigData = require(`../../../networks/${network}.json`);
+const { bnbUnsigned } = require('../utils/web3-utils');
+
+(async () => {
+
+  const xvsStoreAddress = contractConfigData.Contracts.XVSStore;
+  const xvsStoreContractInstance = await saddle.getContractAt('XVSStore', xvsStoreAddress);
+  const xvsVaultAddress = contractConfigData.Contracts.XVSVault;
+  const setNewOwnerToXVSStoreTxn = await xvsStoreContractInstance.methods.setNewOwner(xvsVaultAddress).send();
```
Comment (kkirka) [inline script/deploy/xvs-vault/05-set-new-owner-to-xvs-store.js:10]: Shouldn't we use XVSVaultProxy here and in the later steps?

**Comment by kkirka** [comment]
@defistar is this still WIP?

**Comment by defistar** [comment]
> @defistar is this still WIP?

no its completed, removed WIP

**Comment by kkirka** [comment]
Can you do `git rebase -i --onto master <parent>` pls? Just so that there are no "fix" and "revert" commits. Maybe just sqashing everythig is also an option, use your judgement, but let's make the history neater, and then merge :)

**Pull request closed without merging**